src/plib/views/scripts/index/index.phtml

```diff
@@ -3,4 +3,33 @@
 
 echo '<p>' . \pm_Locale::lmsg('pageDescription') . '</p>';
 
+echo '<div id="piechart" style="width: 800px; height: 400px;"></div>';
+
 echo $this->renderList($this->list);
+
+?>
+
+<script>
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawDiskspaceUsageChart);
+
+function drawDiskspaceUsageChart()
+{
+    var data = new google.visualization.DataTable();
+
+    data.addColumn('string', 'Name');
```

Comment:
localization for titles

Reply:
The chart type we are using do not display titles (axis), so no need to translate these.